js/core/utils/dom.js

```diff
@@ -198,6 +198,57 @@ setPublicElementWrapper(function(element) {
     return element && element.get(0);
 });
 
+var calculateLimitHeight = function(value, container, offset, defaultValue) {
```

Comment:
Why do you place these methods in the `domUtils` instead of `sizeUtils`?

Reply:
we need import core/renderer in these methods, but core/renderer_base.js use size and style utils